ourpackage/andaya.py

```diff
@@ -0,0 +1,64 @@
+import os 
+
+def gener():
+    while True:
+        os.system('cls')
+        print("================= Andaya's Menu ==================")
+        print("=                                                =")
+        print("= Hello Everyone! My name is Gener A. Andaya Jr. =")
+        print("=                                                =")
+        print("= 1. My basic info.                              =")
+        print("= 2. My personal goals.                          =")
+        print("= 3. Comment from Barcelos.                      =")
+        print("= 4. Comment from Veslino.                       =")
+        print("= 5. Comment from Consultado.                    =")
+        print("= 6. Comment from teammate 4.                    =")
```

Comment:
I would like to add a comment into your code - Piadozo

Reply:
You have done it.